mountinfo/mounted_linux_test.go

```diff
@@ -174,10 +174,20 @@ func cleanupMounts(t *testing.T, dir string, mounts []string) {
 	}
 }
 
-func TestMountedBy(t *testing.T) {
+func requireOpenat2(t *testing.T) {
+	t.Helper()
 	if os.Getuid() != 0 {
 		t.Skip("requires root")
 	}
+	fd, err := unix.Openat2(unix.AT_FDCWD, ".", &unix.OpenHow{Flags: unix.O_RDONLY})
+	if err != nil {
+		t.Skipf("openat2: %v (old kernel? need Linux 5.6+)", err)
+	}
+	unix.Close(fd)
+}
+
+func TestMountedBy(t *testing.T) {
+	requireOpenat2(t)
```

Comment:
Do you think it would make sense to only skip the parts that require openat2, instead of the whole test? https://github.com/moby/sys/pull/67

Reply:
I definitely had openat2 way in mind when writing this, mostly added mountinfo to test my test :) 

But yeah, when I look at it, it makes some sense.

Reply:
But as we already test in on an openat2-capable system that should be sufficient.

IOW it's up to you whether you want to update #67 to run this test partially or just close it.